code/modules/spells/spell_types/summonitem.dm

```diff
@@ -41,9 +41,10 @@
 					message = "<span class='warning'>You must hold the desired item in your hands to mark it for recall!</span>"
 
 		else if(marked_item && (marked_item in hand_items)) //unlinking item to the spell
-			message = "<span class='notice'>You remove the mark on [marked_item] to use elsewhere.</span>"
-			name = "Instant Summons"
-			marked_item = 		null
+			if(alert(user,"Unlink the item?",,"Yes","No") == "Yes" && (marked_item in hand_items) && isliving(user))
```

Comment:
what's to keep someone from bringing this alert up early, moving it to the side of the screen, and using the spell as normal until they need to quickly unmark an item while the spell's on cooldown?

Reply:
It’s a 10 second cooldown, seems pointless at best.

Reply:
fix this

Reply:
Since its an alert and the proc sleeps until its interacted with, you also need to re-check all the previous checks made to make sure there is in fact an item to do this to and that it is still in hand etc.

Reply:
I’ll do this today